### ISSUE TITLE
ConvFit Elastic line has incorrect plotting for OSIRIS data

### ISSUE THREAD

**Issue by ElliotAOram** [mantidproject/mantid#13857]
Franz reports that:

``` 
In conv fit when we chose a _red file for the resolution, conv fit refused to fit a delta func for the elastic line
It only fits the quens Lorentzian.
Just to explain: convfit does not show an error. 
Only the fit values are completely wrong.
```

**Comment by SpencerHowells** [comment]
This is probably cured by 14211 - adding shift to the Delta function.

**Comment by ElliotAOram** [comment]
As Spencer has pointed out, this issue has been resolved with the changes made in #14211
